packages/react/src/themes/teams/components/Menu/menuVariables.ts

```diff
@@ -1,89 +1,103 @@
-  focusedBackgroundColor: string
-
-  hoverBackgroundColor: string
 
-  activeColor: string
-  activeBackgroundColor: string
-  iconOnlyActiveColor: string
+  backgroundColor: string
+  backgroundColorFocus: string
+  backgroundColorHover: string
+  backgroundColorActive: string
 
-  primaryActiveColor: string
-  primaryActiveBackgroundColor: string
-  primaryActiveBorderColor: string
+  borderColor: string
+  borderColorHover: string
+  borderColorActive: string
+  borderColorFocus: string
 
-  primaryFocusedColor: string
-  primaryFocusedBackgroundColor: string
+  outlineColorFocus: string
+  colorActive: string
+  iconOnlyColorActive: string
 
-  primaryBorderColor: string
-  primaryHoverBorderColor: string
-  primaryUnderlinedBorderColor: string
+  colorFocus: string
+  underlinedBorderColor: string
 
-  disabledColor: string
+  colorDisabled: string
   lineHeightBase: string
   horizontalPadding: string
 
   verticalBackgroundColor: string
   verticalItemPadding: string
   verticalBoxShadow: string
   verticalDividerMargin: string
-  verticalItemBorder: string
+  verticalItemBorderWidth: string
+  verticalItemBorderColor: string
+  verticalPointingBorderColor: string
+  verticalBackgroundColorFocus: string
 
   pointingIndicatorBackgroundColor: string
 
   underlinedBottomBorderWidth: string
+  primaryBorderColor: string
 
   dividerHeight: string
   borderWidth: string
 }
 
 export default (siteVars: any): MenuVariables => {
   return {
-    color: siteVars.gray02,
-    borderColor: siteVars.gray08,
-    verticalBorderColor: siteVars.gray08,
-
-    focusedBorder: `solid ${pxToRem(1)} ${siteVars.colors.white}`,
-    focusedOutline: `solid ${pxToRem(1)} ${siteVars.colors.black}`,
-    focusedBackgroundColor: siteVars.gray09,
-
-    hoverBackgroundColor: siteVars.gray14,
-
-    activeColor: siteVars.colors.black,
-    activeBackgroundColor: siteVars.gray10,
-    iconOnlyActiveColor: siteVars.colors.primary[500],
-
-    primaryActiveColor: siteVars.colors.white,
-    primaryActiveBackgroundColor: siteVars.brand08,
-    primaryActiveBorderColor: siteVars.colors.primary[500],
-
-    primaryFocusedColor: siteVars.colors.white,
-    primaryFocusedBackgroundColor: siteVars.colors.primary[200],
-
-    primaryBorderColor: siteVars.gray10,
-    primaryHoverBorderColor: siteVars.gray08,
-    primaryUnderlinedBorderColor: siteVars.gray08,
+    colorScheme: extendColorScheme(siteVars.colorScheme, {
+      default: {
+        borderActive: siteVars.colors.grey[600],
+      },
+      brand: {
+        foregroundHover: siteVars.colors.white,
+        backgroundHover: siteVars.colors.brand[300],
+        foregroundActive: siteVars.colors.white,
+        borderActive: siteVars.colors.brand[600],
+        backgroundActive1: siteVars.colors.brand[500], // it's 600 in the color scheme
+        foregroundFocus: siteVars.colors.white,
+        backgroundFocus1: siteVars.colors.brand[300],
+      },
+    }),
+    color: siteVars.colors.grey[500],
```

Comment:
Is there a plan for future refactoring of the color palette/variables?

I am confused with which variable is used for what and when:
- `colorScheme.foreground` is never used, we use `variables.color`
- for active primary item background goes from color scheme (`colorsScheme.backgroundActive1`) and can be overriden by variable (`variables.backgroundColorActive`), color goes from color scheme (`colorScheme.foregroundActive`) but cannot be overridden.
- `v.backgroundColorActive || colorScheme.backgroundActive1` and the variable defaults to `undefined`, but `v.underlinedBorderColor || colorScheme.backgroundActive1`  and the variable is set?

Reply:
Yep, there is a plan for that, but if we were about to do that now, it would took a long time. We will go component by component, and we will update the styles to use the colorScheme values. Usually, they will be used in the following way:
```
{
  root: ({variables}) => {
    const colors = getColorScheme(variables.colorScheme, color, primary);
    return {
      ...,
      color: v.color || colors.foregroundColor,
    }
  },
  ...
}
```

The variables would default to undefined, and we need to come up with a way of how to show it in the docs.

For colors that should never change (even if different color or primary prop is provided, we can have direct value int he variable).